Makefile

```diff
@@ -717,6 +716,12 @@ git-gaia-node-modules: gaia_node_modules.revision
 	fi
 	(cd "$(NODE_MODULES_SRC)" && git fetch && git reset --hard "$(NODE_MODULES_REV)" )
 
+npm-cache:
```

Comment:
Yes, we definitely can. We have to be OK with the project's package.json being responsible for 'rebuild'ing any modules that need that (for modules that have a postinstall step that does weird stuff... right now that's just marionette-js-runner and I don't expect more). Sound good? I have it working locally already.